tokio-trace-fmt/src/filter.rs

```diff
@@ -356,6 +368,46 @@ mod tests {
         assert!(interest.is_always());
     }
 
+    #[test]
+    fn callsite_enabled_includes_span_directive_field() {
```

Comment:
We probably ought to have tests for directives with multiple fields (`[mySpan{foo=1 bar=5}]...`) and for field names but no values (`[mySpan{foo}]...`)..

Reply:
done!